src/index.css

```diff
@@ -55,54 +55,122 @@ a:-webkit-any-link {
   text-decoration: none;
   color: #9c9c9c;
   cursor: pointer;
-  font-size: 12px;
+  font-family: Montserrat, sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
```

Comment:
As these three lines are getting repeated in different classes. The best practice is to create a new class and place these properties there instead of repeating the properties over and over again. It'll save your CSS lines as well 👍🏻 😄 
#### For Example
```css
.class-name {
  font-stretch: normal;
  font-style: normal;
  line-height: normal;
}
```